### PR TITLE
First version of User Guide

### PR DESCRIPTION
**Pull request by AliceBalfanz**

### CODE REVIEW

File: docs/guide/user-guide.md [hunk 1]
```diff
@@ -1 +1,109 @@
 # User guide
+
+In the user guide you can find a description of all components belonging to
```
Comment (gunbra32) [inline docs/guide/user-guide.md:3]: the ->this

File: docs/guide/user-guide.md [hunk 2]
```diff
@@ -1 +1,109 @@
 # User guide
+
+In the user guide you can find a description of all components belonging to 
+DeepESDL.
+
+There are three components of DeepESDL:
```
Comment (gunbra32) [inline docs/guide/user-guide.md:6]: "The growing set of DeepESDL's general functionalities comprises:"

File: docs/guide/user-guide.md [hunk 3]
```diff
@@ -1 +1,109 @@
 # User guide
+
+In the user guide you can find a description of all components belonging to 
+DeepESDL.
+
+There are three components of DeepESDL: 
+1. [DeepESDL JupyterLab](#deepesdl-jupyterlab)
+2. [DeepESDL xcube viewer](#deepesdl-xcube-viewer)
+3. [DeepESDL datacube resources](#deepesdl-datacube-resources)
```
Comment (gunbra32) [inline docs/guide/user-guide.md:9]: 4. [DeepESDL data cube generation] (# deepesdl-datacube-generation)

Here, we describe our general approach of data cube generation and refer to i) example notebooks for ad hoc generation by users and ii) recipes for published, reproducible cubes and the cube specification.

File: docs/guide/user-guide.md [hunk 4]
```diff
@@ -1 +1,109 @@
...
+
+
+# DeepESDL JupyterLab
+
+## Basic usage
+
+This section provides a brief introduction for users to the basic features of
+the JupyterLab environment as deployed on the DeepESDL system.
```
Comment (gunbra32) [inline docs/guide/user-guide.md:17]: as offered by DeepESDL.

File: docs/guide/user-guide.md [hunk 5]
```diff
@@ -1 +1,109 @@
...
+
+## Further information
+
+ - The [JupyterLab documentation](https://jupyterlab.readthedocs.io/):
+   an in-depth user guide for the JupyterLab interface.
+ - [How to Use JupyterLab](https://www.youtube.com/watch?v=A5YyoCKxEOU):
+   a short introductory video tutorial.
+ - [Markdown cells](https://jupyter-notebook.readthedocs.io/en/stable/examples/Notebook/Working%20With%20Markdown%20Cells.html),
+   a guide to writing Markdown in Jupyter notebooks.
+ - The [xcube documentation](https://xcube.readthedocs.io/): user
```
Comment (gunbra32) [inline docs/guide/user-guide.md:108]: memo: xcube documentation to be updated.

File: docs/guide/user-guide.md [hunk 6]
```diff
@@ -1 +1,109 @@
...
+For more in-depth documentation on the various components, see the links in the
+[‘Further information’ section](#further-information).
+
+### Logging in and starting the JupyterLab profile
+
+To use the DeepESDL JupyterLab environment, navigate to
+<https://deep.earthsystemdatalab.net/> with a web browser (a recent version of
+Firefox, Chrome, or Safari is recommended).
+
+DeepESDL uses a GitHub to authenticate, so if you are already registered as a
```
Comment (gunbra32) [inline docs/guide/user-guide.md:27]: DeepESDL uses GitHub to authenticate
User must be registered first. We thus need a sentence lie: Before first usage, we will have to register you with the system. To this end, we kindly ask you to duly compete the registration dialogue (link)/write as an email/...

File: docs/guide/user-guide.md [hunk 7]
```diff
@@ -1 +1,109 @@
...
+<https://deep.earthsystemdatalab.net/> with a web browser (a recent version of
+Firefox, Chrome, or Safari is recommended).
+
+DeepESDL uses a GitHub to authenticate, so if you are already registered as a 
+DeepESDL user, please use your GitHub account to log in. 
+If your Jupyter server is not already running, you may be presented
+with a menu of user JupyterLab profiles to use for your session; there might be 
+one or more JupyterLab profiles to choose from, depending on the computational 
+resources needs of your team. Please select an adequate JupyterLab profiles for 
+your current task; it might not always require the strongest computational
```
Comment (gunbra32) [inline docs/guide/user-guide.md:33]: strongest -> all available computational resources.

File: docs/guide/user-guide.md [hunk 8]
```diff
@@ -1 +1,109 @@
...
+
+DeepESDL uses a GitHub to authenticate, so if you are already registered as a 
+DeepESDL user, please use your GitHub account to log in. 
+If your Jupyter server is not already running, you may be presented
+with a menu of user JupyterLab profiles to use for your session; there might be 
+one or more JupyterLab profiles to choose from, depending on the computational 
+resources needs of your team. Please select an adequate JupyterLab profiles for 
+your current task; it might not always require the strongest computational 
+resources available.
+After choosing your environment, you will see a progress bar appear for a few
```
Comment (gunbra32) [inline docs/guide/user-guide.md:35]: appearing

File: docs/guide/user-guide.md [hunk 9]
```diff
@@ -1 +1,109 @@
...
+The JupyterLab interface will then appear in your web browser, ready for
+use.
+
+
+### Choosing a JupyterLab profile
+
+If you don't already have a running DeepESDL Jupyter session, you will be 
+presented with a list of one or more available user JupyterLab profiles when 
+you log in. 
+A user JupyterLab profiles provides a particular combination of pre-installed
```
Comment (gunbra32) [inline docs/guide/user-guide.md:46]: A JupyterLab profile

File: docs/guide/user-guide.md [hunk 10]
```diff
@@ -1 +1,109 @@
...
+
+### Choosing a JupyterLab profile
+
+If you don't already have a running DeepESDL Jupyter session, you will be 
+presented with a list of one or more available user JupyterLab profiles when 
+you log in. 
+A user JupyterLab profiles provides a particular combination of pre-installed 
+package versions and resource settings. 
+
+If you have already started your session and need to change JupyterLab profiles,
```
Comment (gunbra32) [inline docs/guide/user-guide.md:49]: profile

File: docs/guide/user-guide.md [hunk 11]
```diff
@@ -1 +1,109 @@
...
+you have logged out, but will eventually be terminated due to inactivity. 
+If you wish to stop your session explicitly,
+you can use the hub control panel as described in the 
+[Choosing a JupyterLab profile](#choosing-a-jupyterlab-profile) section above.
+
+
+
+# DeepESDL xcube viewer
+
+The DeepESDL xcube viewer is reachable via
```
Comment (gunbra32) [inline docs/guide/user-guide.md:69]: reachable at

File: docs/guide/user-guide.md [hunk 12]
```diff
@@ -1 +1,109 @@
...
+If you wish to stop your session explicitly,
+you can use the hub control panel as described in the 
+[Choosing a JupyterLab profile](#choosing-a-jupyterlab-profile) section above.
+
+
+
+# DeepESDL xcube viewer
+
+The DeepESDL xcube viewer is reachable via 
+https://viewer.earthsystemdatalab.net/ .
```
Comment (gunbra32) [inline docs/guide/user-guide.md:70]: https://viewer.earthsystemdatalab.net

File: docs/guide/user-guide.md [hunk 13]
```diff
@@ -1 +1,109 @@
...
+
+![img.png](../img/xcube-viewer.png)
+
+The viewer currently shows public datasets, and later will allow to see
```
Comment (gunbra32) [inline docs/guide/user-guide.md:74]: contains public datasets only, but will later also provide user cubes when logged in.

File: docs/guide/user-guide.md [hunk 14]
```diff
@@ -1 +1,109 @@
...
+# DeepESDL xcube viewer
+
+The DeepESDL xcube viewer is reachable via 
+https://viewer.earthsystemdatalab.net/ .
+
+![img.png](../img/xcube-viewer.png)
+
+The viewer currently shows public datasets, and later will allow to see 
+team/user cubes when logged in. The login will be via GitHub as for the DeepESDL
+JupyterLab.
```
Comment (gunbra32) [inline docs/guide/user-guide.md:76]: The login will be the same as for the DeepESDL JupyterLab.

File: docs/guide/user-guide.md [hunk 15]
```diff
@@ -1 +1,109 @@
...
+https://xcube.readthedocs.io/en/latest/viewer.html#functionality
+
+
+# DeepESDL datacube resources
+
+DeepESDL provides a growing list of relevant variables for Earth System Science. 
+Most of them have been derived from Earth Observation, but the compilation also 
+includes model or re-analysis data if deemed useful.
+DeepESDL is very grateful to all data owners for kindly providing the data sets 
+free of charge. They come without any warranty, neither from the owners,
```
Comment (gunbra32) [inline docs/guide/user-guide.md:88]: and allowing us to process, and redistribute them,

File: docs/guide/user-guide.md [hunk 16]
```diff
@@ -1 +1,109 @@
...
+
+
+# DeepESDL datacube resources
+
+DeepESDL provides a growing list of relevant variables for Earth System Science. 
+Most of them have been derived from Earth Observation, but the compilation also 
+includes model or re-analysis data if deemed useful.
+DeepESDL is very grateful to all data owners for kindly providing the data sets 
+free of charge. They come without any warranty, neither from the owners, 
+the DeepESDL team or ESA.
```
Comment (gunbra32) [inline docs/guide/user-guide.md:89]: All datacubes generated and distributed by DeepESDL come without any warranty, neither from the owners, from the DeepESDL, nor form ESA.

File: docs/guide/user-guide.md [hunk 17]
```diff
@@ -1 +1,109 @@
...
+
+At ingestion into the DeepESDL, all data sets are transformed in space and time
```
Comment (gunbra32) [inline docs/guide/user-guide.md:91]: data sets are typically transformed ...

File: docs/guide/user-guide.md [hunk 18]
```diff
@@ -1 +1,109 @@
...
+DeepESDL provides a growing list of relevant variables for Earth System Science. 
+Most of them have been derived from Earth Observation, but the compilation also 
+includes model or re-analysis data if deemed useful.
+DeepESDL is very grateful to all data owners for kindly providing the data sets 
+free of charge. They come without any warranty, neither from the owners, 
+the DeepESDL team or ESA.
+
+At ingestion into the DeepESDL, all data sets are transformed in space and time 
+to fit to the common grid of the data cube, a process that necessarily modifies 
+the original. If you are looking for the original data, please follow the links
```
Comment (gunbra32) [inline docs/guide/user-guide.md:93]: the original data.

File: docs/guide/user-guide.md [hunk 19]
```diff
@@ -1 +1,109 @@
...
+the DeepESDL team or ESA.
+
+At ingestion into the DeepESDL, all data sets are transformed in space and time 
+to fit to the common grid of the data cube, a process that necessarily modifies 
+the original. If you are looking for the original data, please follow the links 
+within the dataset attributes for each variable and contact the data owners.
+
+To access the documentation of available datasets, please have a look in the 
+[datasets section](../datasets/datasets.md).
+
```
Comment (gunbra32) [inline docs/guide/user-guide.md:98]: add 4 section here (see above for details).

**Review by gunbra32 — CHANGES_REQUESTED** [review]

File: docs/guide/user-guide.md (hunk 3 above)
Comment (forman) [inline docs/guide/user-guide.md:9]:
```suggestion
3. [DeepESDL datacubes](#deepesdl-datacube-resources)
```

To me _Datacube resources_  sounds like resources attached to or required for datacubes.

File: docs/guide/user-guide.md [hunk 20]
```diff
@@ -1 +1,109 @@
...
+The viewer currently shows public datasets, and later will allow to see 
+team/user cubes when logged in. The login will be via GitHub as for the DeepESDL
+JupyterLab. 
+
+For the functionality of the viewer, please refer to the documentation:
+https://xcube.readthedocs.io/en/latest/viewer.html#functionality
+
+
+# DeepESDL datacube resources
```
Comment (forman) [inline docs/guide/user-guide.md:82]:
```suggestion
# DeepESDL datacubes
```

File: docs/guide/user-guide.md [hunk 21]
```diff
@@ -1 +1,109 @@
...
+
+
+# DeepESDL JupyterLab
+
+## Basic usage
+
+This section provides a brief introduction for users to the basic features of
+the JupyterLab environment as deployed on the DeepESDL system. 
+For more in-depth documentation on the various components, see the links in the
+[‘Further information’ section](#further-information).
```
Comment (forman) [inline docs/guide/user-guide.md:19]:
```suggestion
section [Further Information](#further-information).
```

File: docs/guide/user-guide.md (hunk 1 above)
Comment (forman) [inline docs/guide/user-guide.md:3]:
```suggestion
This user guide helps you getting started with the main components of 
```

File: docs/guide/user-guide.md [hunk 22]
```diff
@@ -1 +1,109 @@
...
+To use the DeepESDL JupyterLab environment, navigate to
+<https://deep.earthsystemdatalab.net/> with a web browser (a recent version of
+Firefox, Chrome, or Safari is recommended).
+
+DeepESDL uses a GitHub to authenticate, so if you are already registered as a 
+DeepESDL user, please use your GitHub account to log in. 
+If your Jupyter server is not already running, you may be presented
+with a menu of user JupyterLab profiles to use for your session; there might be 
+one or more JupyterLab profiles to choose from, depending on the computational 
+resources needs of your team. Please select an adequate JupyterLab profiles for
```
Comment (forman) [inline docs/guide/user-guide.md:32]:
```suggestion
resources needs of your team. Please select a suitable profile for 
```

File: docs/guide/user-guide.md [hunk 23]
```diff
@@ -1 +1,109 @@
...
+resources needs of your team. Please select an adequate JupyterLab profiles for 
+your current task; it might not always require the strongest computational 
+resources available.
+After choosing your environment, you will see a progress bar appear for a few 
+seconds while it is started for you. 
+The JupyterLab interface will then appear in your web browser, ready for
+use.
+
+
+### Choosing a JupyterLab profile
```
Comment (forman) [inline docs/guide/user-guide.md:41]: Note, you already discussed above which profile to choose.

File: docs/guide/user-guide.md [hunk 24]
```diff
@@ -1 +1,109 @@
...
+
+If you don't already have a running DeepESDL Jupyter session, you will be 
+presented with a list of one or more available user JupyterLab profiles when 
+you log in. 
+A user JupyterLab profiles provides a particular combination of pre-installed 
+package versions and resource settings. 
+
+If you have already started your session and need to change JupyterLab profiles, 
+you can do this by selecting ‘Hub control panel’ from the ‘File’ menu within
```
Comment (forman) [inline docs/guide/user-guide.md:50]:
```suggestion
you can do this by selecting *Hub control panel* from the *File* menu within
```

To me, putting phrases in quotation marks is indicating either quotes, code string literals, verbatim speech, or ironic meaning. Therefore I prefer highlighting of UI elements via font style.

File: docs/guide/user-guide.md [hunk 25]
```diff
@@ -1 +1,109 @@
...
+https://viewer.earthsystemdatalab.net/ .
+
+![img.png](../img/xcube-viewer.png)
+
+The viewer currently shows public datasets, and later will allow to see 
+team/user cubes when logged in. The login will be via GitHub as for the DeepESDL
+JupyterLab. 
+
+For the functionality of the viewer, please refer to the documentation:
+https://xcube.readthedocs.io/en/latest/viewer.html#functionality
```
Comment (forman) [inline docs/guide/user-guide.md:79]:
```suggestion
For a more detailed description of the viewer functionality, please refer to a dedicated section in the
[xcube documentation](https://xcube.readthedocs.io/en/latest/viewer.html#functionality).
```

File: docs/guide/user-guide.md [hunk 26]
```diff
@@ -1 +1,255 @@
...
+
+### Choosing a JupyterLab profile
+
+If you don't already have a running DeepESDL Jupyter session, you will be 
+presented with a list of one or more available user JupyterLab profiles when 
+you log in. 
+A user JupyterLab profile provides a particular combination of pre-installed 
+package versions and resource settings. 
+
+If you have already started your session and need to change JupyterLab profiles,
```
Comment (AliceBalfanz) [inline docs/guide/user-guide.md:50]:
```suggestion
If you have already started your session and need to change the JupyterLab profile, 
```

File: docs/guide/user-guide.md [hunk 27]
```diff
@@ -1 +1,249 @@
...
+
+# DeepESDL xcube viewer
+
+The DeepESDL xcube viewer is reachable at 
+https://viewer.earthsystemdatalab.net .
+
+![img.png](../img/xcube-viewer.png)
+
+The viewer contains public datasets only, but will later also provide user/team 
+cubes when logged in. TThe login will be the same as for the DeepESDL JupyterLab.
```
Comment (AliceBalfanz) [inline docs/guide/user-guide.md:86]:
```suggestion
cubes when logged in. The login will be the same as for the DeepESDL JupyterLab.
```

File: docs/guide/user-guide.md [hunk 28]
```diff
@@ -1 +1,249 @@
...
+variables, a GeoJSON FeatureCollection may be used instead. To validate the 
+JSON cube specification files, we provide a dedicated online JSON Schema 
+in the DeepESDL dataset-spec GitHub repository:
+ https://github.com/deepesdl/dataset-spec/blob/main/dataset-defs/template.schema.json
+
+### Cube generation recipe
+
+The cube generation recipe ensures, that an existing dataset provided by 
+DeepESDL can be reproduced by following the recipe. Here we describe the 
+recipe structure of the provided datasets within DeepESDL:
```
Comment (AliceBalfanz) [inline docs/guide/user-guide.md:147]:
```suggestion
recipe structure of the provided datasets within DeepESDL.
```

File: docs/guide/user-guide.md [hunk 29]
```diff
@@ -1 +1,249 @@
...
+representation, i.e. they “exists” at a given storage location. That way 
+the data can be “frozen” and can be assigned a version and/or a DOI. In 
+addition, access to static data cubes persisted in cloud object storage 
+usually is fast and scalable. 
+
+However, there are several issues and challenges with static cubes. Data 
+sources of data cubes can become outdated,
+or are simply updated, like it is the case for new EO data observations 
+added to an existing product archive. In such cases, related data cubes should 
+be updated too, hereby creating considerable maintenance effort. In addition,
```
Comment (gunbra32) [inline docs/guide/user-guide.md:191]:
```suggestion
be updated too, hereby creating considerable maintenance effort and risks for the integrity of the data cube. In addition,
```

File: docs/guide/user-guide.md [hunk 30]
```diff
@@ -1 +1,249 @@
...
+the generation of static data cubes is in many cases a plain duplication of 
+data that is actually defined and described elsewhere. The data cube must 
+ensure to stay in sync with original data sources and metadata.
+Finally, static cubes can only satisfy requirements of one user or use 
+casein an optimal way. By definition, they do not allow for streamlined and
```
Comment (gunbra32) [inline docs/guide/user-guide.md:196]:
```suggestion
case in an optimal way. By definition, they do not allow for streamlined and 
```

**Review by gunbra32 — CHANGES_REQUESTED** [review]

File: docs/guide/user-guide.md [hunk 31]
```diff
@@ -1 +1,249 @@
...
+needs to be e.g. downloaded beforehand or other preprocessing steps are 
+performed, then the cube generation recipe approach is the suitable one. 
+
+## Cube generation recipe approach for static data cubes
+
+### Cube specification format
+For each data cube, a unique specification is created that describes 
+the cubes spatio-temporal dimensions, resolutions, coverages, and the data 
+sources used to generate the cube’s target data variables. To specify each 
+cube, a dedicated JSON format is created that fully describes the target
```
[the remark this answers is not in the document]
Comment (AliceBalfanz) [reply docs/guide/user-guide.md:131]: Further below in the git-repo link

**Pull request merged**